libmscore/select.h

```diff
@@ -61,6 +63,8 @@ struct NotePattern {
       int staffStart;
       int staffEnd; // exclusive
       int voice;
+      Fraction beat {0,0};
+      const Measure* measure = nullptr;
```

Comment:
Aside: it probably wouldn't hurt to have Element* also be initialized to nullptr so that the Element = 0 codes in the other functions could be eliminated, but that's not too important. It reads funny to have one pointer initalized to nullptr and the next one not though...

Reply:
Why not just do it then?

Reply:
Oh, more than likely a trend toward "minimal intrusiveness".